source/dub/dub.d

```diff
@@ -76,7 +77,12 @@ enum defaultRegistryURL = "http://code.dlang.org/";
 PackageSupplier[] defaultPackageSuppliers()
 {
 	logDiagnostic("Using dub registry url '%s'", defaultRegistryURL);
-	return [new RegistryPackageSupplier(URL(defaultRegistryURL))];
+	return [
+		new FallbackPackageSupplier(
+			new RegistryPackageSupplier(URL(defaultRegistryURL)),
+			new RegistryPackageSupplier(URL(fallbackRegistryURL))
+		)
+	];
```

Comment:
Why not simply return an array of the two? The intent is already that later array entries act as fallbacks to the earlier ones.